### PR TITLE
add: 404

### LINKED ISSUE #233: Implement a 404 page
### Description

If you navigate to unknown url we should redirect to a 404 page.

### Screenshots

image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by kymppi**
fixes #233 

I hope this PR is awesome. I tried to create a cool design :tada:

image
image

### CODE REVIEW

File: backend/src/app.controller.ts [hunk 1]
```diff
@@ -20,13 +20,22 @@ export class AppController {
     return;
   }
 
+  @Get()
+  @Render('unknown')
+  unknown() {
+    console.log('here');
```
Comment (GuillemPM) [inline backend/src/app.controller.ts:26]: Maybe it's better to don't show any log in console
```suggestion
  
```

File: frontend/src/routes/unknown/index.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          </a>
+        </div>
+      </div>
+      <Waves />
+    </section>
+  );
+});
+
+export const head: DocumentHead = {
+  title: 'The FREE Open-Source URL Shortener | Reduced.to',
```
Comment (GuillemPM) [inline frontend/src/routes/unknown/index.tsx:42]: Let's customize the title for this page :)
```suggestion
  title: '🤷‍♀️ Page not found | Reduced.to',
```

**Review by GuillemPM — CHANGES_REQUESTED** [review]
Hey, the page looks really good, would you mind to apply the requested changes?
Thanks!! 🔥

**Comment by kymppi** [comment]
> Hey, the page looks really good, would you mind to apply the requested changes? Thanks!! fire

Fixed :rocket:

**Review by GuillemPM — APPROVED** [review]

**Comment by kymppi** [comment]
Hmmm, ill go run prettier now...

**Comment by GuillemPM** [comment]
> Hmmm, ill go run prettier now...

It should be handled by Husky on pre-commit hook, the problem was probably modifying the files from Github reviews. Thanks!

**Comment by kymppi** [comment]
Yeah, at first there weren't any problems but after I pulled the changes It showed them ;P

**Comment by GuillemPM** [comment]
All passing, I'll merge this, thank you so much!! 🚀

**Comment by kymppi** [comment]
:+1:

**Pull request merged**